legacy/endpoints/login/login.yml

```diff
@@ -51,6 +51,8 @@ paths:
   /api/user/{user_id}/login:
     post:
       summary: User login
+      description: >-
+        The logged user can call this API with cookie and get a token. Reference in https://confluence.aminocom.com/display/MAPI/Cookie-token+exchange
```

Comment:
This implies this specific case is what the endpoint is used for. I would suggest

> The client uses this API to authenticate with the service and retrieve session parameters and access tokens used for authenticating with APIs.
A logged in client may call this API to retrieve tokens without supplying the authorization parameters, see https://confluence.aminocom.com/display/MAPI/Cookie-token+exchange